src/test/java/org/simplify4u/plugins/keyserver/PGPKeysServerClientTest.java

```diff
@@ -1,45 +1,34 @@
 package org.simplify4u.plugins.keyserver;
 
-import static org.simplify4u.plugins.utils.ProxyUtil.makeMavenProxy;
-import java.io.IOException;
-import java.net.URI;
-import java.net.URISyntaxException;
```

Comment:
imports order

Reply:
Reverted, it's the auto formatting on intellij when I do the import. What config have you got on your IDE?

Reply:
I will  chek my current configuration according to default configuration in InteliJ, maybe I was changed something in this section.
Thank for hint - it will be easier for other to have default configuration in this case.